src/domain/reservation/manage/page/ManageReservationsPage.js

```diff
@@ -176,4 +209,13 @@ class ManageReservationsPage extends React.Component {
 }
 
 export const UnwrappedManageReservationsPage = injectT(ManageReservationsPage);
-export default withRouter(UnwrappedManageReservationsPage);
+
+const mapDispatchToProps = (dispatch) => {
+  const actionCreators = {
+    editReservation: selectReservationToEdit
```

Comment:
Not sure why we still need this!

Reply:
To populate the timeslot. I have a todo to remove them in the future